package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tableau-mcp",
   "description": "A MCP server for Tableau, providing a suite of tools that will make it easier for developers to build AI-applications that integrate with Tableau.",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
This can probably just be a `patch` update. Please update the version in the package-lock file too.